shell/packages/sandstorm-db/profile.js

```diff
@@ -211,10 +212,10 @@ SandstormDb.fillInProfileDefaults = function (user) {
     const setting = Settings.findOne({ _id: "ldapNameField" });
     const key = setting ? setting.value : "";
     profile.name = profile.name || user.services.ldap.rawAttrs[key] || "Name Unknown";
-    profile.handle = profile.handle || filterHandle(profile.name);
+    profile.handle = profile.handle || user.services.ldap.username;
   } else if (profile.service === "saml") {
     profile.name = profile.name || user.services.saml.displayName || "Name Unknown";
-    profile.handle = profile.handle || filterHandle(profile.name);
+    profile.handle = profile.handle || user.services.saml.email.split("@")[0];
```

Comment:
This should use the `emailToHandle` function that's already defined elsewhere in this file.